pymc3/sampling.py

```diff
@@ -585,7 +576,7 @@ def sample(
                     UserWarning,
                 )
             _print_step_hierarchy(step)
-            trace = _sample_population(**sample_args, parallelize=cores > 1)
+            trace = _sample_population(parallelize=cores > 1, **sample_args)
```

Comment:
this is just for `mypy`'s sake

Reply:
I like type annotations.
And while I don't like how a lot of it is done through these weird comments, if this is the way...